samples/pytorch/client.sh

```diff
@@ -0,0 +1,16 @@
+HOST=localhost
```

Comment:
Can the client support command line parameter to send a jpg file specified on the command line to the inference service?